packages/framework/src/Model/Product/Elasticsearch/Scope/ProductExportScopeConfig.php

```diff
@@ -142,6 +144,12 @@ protected function loadProductExportScopeRules(): void
             self::PRECONDITION_VISIBILITY_RECALCULATION,
             self::PRECONDITION_SELLING_DENIED_RECALCULATION,
         ]);
+        $this->addNewExportScopeRule(self::SCOPE_DOMAIN_URL, [
```

Comment:
I would rather add the new fields to `SCOPE_URL` - otherwise it might be confusing which one should be used, or? I see also `HREFLANG_LINKS` in that scope that is missing here but I think it should be there as well, right?

Reply:
After rework I have updated this scope to only include descriptions